<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTPTechGroup::ModelerIde::FilterPropWidget</class>
 <widget class="QWidget" name="RTPTechGroup::ModelerIde::FilterPropWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPropTool">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="modeleride.qrc">
         <normaloff>:/add</normaloff>:/add</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonEdit">
       <property name="text">
        <string>Изменить</string>
       </property>
       <property name="icon">
        <iconset resource="modeleride.qrc">
         <normaloff>:/edit</normaloff>:/edit</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDel">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="modeleride.qrc">
         <normaloff>:/remove</normaloff>:/remove</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxProp">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditSrcClass"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAlias">
        <property name="text">
         <string>Псевдоним:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Имя фильтра:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelDestClass">
        <property name="text">
         <string>Второй класс:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelSrcClass">
        <property name="text">
         <string>Первый класс:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditDirectDesc">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditInverseDesc">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditAlias">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDirectDesc">
        <property name="text">
         <string>Описание прямое:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="RTPTechGroup::Widgets::TreeComboBox" name="comboBoxDestClass"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelInverseDesc">
        <property name="text">
         <string>Описание обратное:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCondition">
     <property name="title">
      <string>Условие фильтра</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutConditionButton">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonCondAdd">
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/add</normaloff>:/add</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonCondDel">
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/remove</normaloff>:/remove</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerCondition">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="treeViewCondition">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPropSave">
     <item>
      <spacer name="horizontalSpacerRightProp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPropSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPropCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerLeftProp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RTPTechGroup::Widgets::TreeComboBox</class>
   <extends>QComboBox</extends>
   <header>treecombobox/treecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="modeleride.qrc"/>
 </resources>
 <connections/>
</ui>
